### PR TITLE
research-design.tex

### PR DESCRIPTION
**Pull request by mariaruth**
Update research design chapter to frame as appendix

### CODE REVIEW

**Comment by mariaruth** [comment]
@bbdaniels, re: side notes, i'm not sure best thing to do with the below paragraph, i see two options:
1. move all the references to a wiki page on causal inference
2. change these to proper citations rather than just URLs (since they are books i assume that should work)
Which would be more consistent with the changes you've been making to other chapters? 

There are several resources that provide more or less mathematically intensive
approaches to understanding how various methods do this.
\textit{Impact Evaluation in Practice} is a strong general guide to these methods.\sidenote{
  \url{https://www.worldbank.org/en/programs/sief-trust-fund/publication/impact-evaluation-in-practice}}
\textit{Causal Inference} and \textit{Causal Inference: The Mixtape}
provides more detailed mathematical approaches to the tools.\sidenote{
  \url{https://www.hsph.harvard.edu/miguel-hernan/causal-inference-book}
  \\ \noindent \url{http://scunning.com/cunningham_mixtape.pdf}}
\textit{Mostly Harmless Econometrics} and \textit{Mastering Metrics}
are excellent resources on the statistical principles behind all econometric approaches.\sidenote{
  \url{https://www.researchgate.net/publication/51992844\_Mostly\_Harmless\_Econometrics\_An\_Empiricist's\_Companion}
  \\ \noindent \url{https://assets.press.princeton.edu/chapters/s10363.pdf}}

**Comment by mariaruth** [comment]
@bbdaniels I am assuming that we will leave all Development Impact blog links directly in text? for now i have, but wanted to confirm. a few examples below but there are many more in the appendix. 

Loss of statistical power occurs quickly and is highly nonlinear:
70\% take-up or efficacy doubles the required sample, and 50\% quadruples it.\sidenote{
  \url{https://blogs.worldbank.org/impactevaluations/power-calculations-101-dealing-with-incomplete-take-up}}

Only these details are needed to construct the appropriate estimator:
clustering of the standard errors is required at the level
at which the treatment is assigned to observations,
and variables which were used to stratify the treatment
must be included as controls (in the form of strata fixed effects).\sidenote{
  \url{https://blogs.worldbank.org/impactevaluations/impactevaluations/how-randomize-using-many-baseline-variables-guest-post-thomas-barrios}}

However, controls for balance variables are usually unnecessary in RCTs,
because it is certain that the true data-generating process
has no correlation between the treatment and the balance factors.\sidenote{
  \url{https://blogs.worldbank.org/impactevaluations/should-we-require-balance-t-tests-baseline-observables-randomized-experiments}}

File: appendix/research-design.tex [hunk 1]
```diff
@@ -1,41 +1,45 @@
 %-----------------------------------------------------------------------------------------------
 
 \begin{fullwidth}
-Research design is the process of defining the methods and data
-that will be used to answer a specific research question.
-You don't need to be an expert in research design to do effective data work,
-but it is essential that you understand the design of the study you are working on,
-and how it affects the data work.
+Development Research in Practice focuses on tools, workflows, and practical
```
Comment (bbdaniels) [inline appendix/research-design.tex:4]:
```suggestion
\textit{Development Research in Practice} focuses on tools, workflows, and practical 
```

File: appendix/research-design.tex [hunk 2]
```diff
@@ -1,41 +1,45 @@
...
 \begin{fullwidth}
-Research design is the process of defining the methods and data
-that will be used to answer a specific research question.
-You don't need to be an expert in research design to do effective data work,
-but it is essential that you understand the design of the study you are working on,
-and how it affects the data work.
+Development Research in Practice focuses on tools, workflows, and practical 
+guidance for implementing research projects. 
+While not central to the content of this book, 
+we realize how essential it is for all research team members
```
Comment (bbdaniels) [inline appendix/research-design.tex:7]:
```suggestion
we think it is essential for all research team members
```

File: appendix/research-design.tex [hunk 3]
```diff
@@ -1,41 +1,45 @@
...
+- including field staff and research assistants -
```
Comment (bbdaniels) [inline appendix/research-design.tex:8]:
```suggestion
-- including field staff and research assistants -- 
```

File: appendix/research-design.tex [hunk 4]
```diff
@@ -1,41 +1,45 @@
...
-You don't need to be an expert in research design to do effective data work,
-but it is essential that you understand the design of the study you are working on,
-and how it affects the data work.
+Development Research in Practice focuses on tools, workflows, and practical 
+guidance for implementing research projects. 
+While not central to the content of this book, 
+we realize how essential it is for all research team members
+- including field staff and research assistants - 
+to understand research design, 
+and specifically how that research design impacts data work.
```
Comment (bbdaniels) [inline appendix/research-design.tex:10]:
```suggestion
and specifically how research design choices impact data work. 
```

File: appendix/research-design.tex [hunk 5]
```diff
@@ -1,41 +1,45 @@
...
 many of the most interesting projects occur because people in the field
 recognize the opportunity to implement one of these methods
 in response to an unexpected event.
-Intuitive knowledge of your project's chosen approach will make you
-much more effective at the analytical part of your work.
 
-This chapter first covers causal inference methods.
-Next it discusses how to measure treatment effects and structure data for specific methods,
+This appendix is split into two sections.
+The first covers causal inference methods,
```
Comment (bbdaniels) [inline appendix/research-design.tex:40]:
```suggestion
The first covers causal inference methods
```

File: appendix/research-design.tex [hunk 6]
```diff
@@ -172,13 +174,14 @@ \subsection{Experimental and quasi-experimental research designs}
 if they had not been treated, and it is particularly effective at doing so
 as evidenced by its broad credibility in fields ranging from clinical medicine to development.
 Therefore RCTs are very popular tools for determining the causal impact
-of specific programs or policy interventions.\sidenote{
-  \url{https://www.nobelprize.org/prizes/economic-sciences/2019/ceremony-speech}}
+of specific programs or policy interventions, 
+as evidenced by the awarding of the 2019 Nobel Prize in Economics
+to Abhijit Banerjee, Esther Duflo and Michael Kremer 
+“for their experimental approach to alleviating global poverty”.\cite{nobel2019}
```
Comment (bbdaniels) [inline appendix/research-design.tex:180]:
```suggestion
``for their experimental approach to alleviating global poverty''.\cite{nobel2019}
```

**Review by bbdaniels — CHANGES_REQUESTED** [review]
Minor cleanup. Going to compile now and add any other changes.

**Pull request merged**